framework/src/transfers/Transfer.C

```diff
@@ -38,12 +38,13 @@ Transfer::validParams()
   params += SetupInterface::validParams();
   params.set<ExecFlagEnum>("execute_on", true) = EXEC_TIMESTEP_BEGIN;
 
-  MultiMooseEnum possible_directions(Transfer::possibleDirections());
-  params.addRequiredParam<MultiMooseEnum>(
+  MultiMooseEnum possible_directions(Transfer::possibleDirections(), "");
+  params.addDeprecatedParam<MultiMooseEnum>(
```

Comment:
Looks like this parameter is marked as being set for the purposes of the deprecated case. May have something to do with a blank default value, I dunno. Not super familiar with the inner workings of a moose num.